foxglove/console/client.go

```diff
@@ -195,6 +196,31 @@ func (c *FoxgloveClient) post(
 	return nil
 }
 
+func (c *FoxgloveClient) delete(endpoint string) error {
+	req, err := http.NewRequest(http.MethodDelete, c.baseurl+endpoint, nil)
+	if err != nil {
+		return fmt.Errorf("failed to create request: %w", err)
+	}
+
+	resp, err := c.authed.Do(req)
+	if err != nil {
+		return fmt.Errorf("request failed: %w", err)
+	}
+
+	switch resp.StatusCode {
+	case http.StatusForbidden:
+		return ErrForbidden
+	case http.StatusNotFound:
+		// Warn the user, but proceed as successful
+		fmt.Fprintf(os.Stderr, "Not found. The resource may have already been deleted.")
```

Comment:
will this result in a newline terminated output still?

Reply:
Nope; fixed.